tests/installation/add_maintenance_repos.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Add new add-ons using maintenance test repo URLs.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    my @repos = split(/,/, get_var('MAINT_TEST_REPO'));
+
+    $testapi::distri->get_add_on_product()->confirm_like_additional_add_on();
+
+    my $maintrepo = shift @repos;
+    $testapi::distri->get_add_on_product()->accept_current_media_type_selection();
+    $testapi::distri->get_repository_url()->add_repo({url => $maintrepo});
+    save_screenshot;
+
+    foreach (@repos) {
+        $testapi::distri->get_add_on_product_installation()->add_add_on_product();
+        $testapi::distri->get_add_on_product()->accept_current_media_type_selection();
+        my $repo = $_;
+        my ($repo_id, $addon) = $repo =~ (/(\d+)\/(.*)\//);
+        my $name = $addon . "_" . $repo_id;
+        $testapi::distri->get_repository_url()->add_repo({url => $repo, name => $name});
+        save_screenshot;
```

Comment:
I think we just need an screenshot after line 21 in the whole file, otherwise when there will be 40 repos, it is too much. wdyt?

Reply:
I'm thinking, we could show a screenshot for each repo. This way one can check repo-specific stuff, should they want to. Even check names, too. Plus, it will resemble the previous needle based behavior (only this time we will strictly get 1 screenshot per repo).

If you think we don't need that information, then I think that even no screenshot at all is fine. But I can insert one at line 19. Should I do that?

Reply:
no, it looks very neat as you designed! good choice!